build.sh

```diff
@@ -20,11 +21,28 @@ function exitTrap() {
     fi
 }
 
+function installCore() {
```

Comment:
Similar to my other comment this would be outside the scope of this issue, but it would be nice to have `installCore` as a separate argument for the build script

Reply:
nice idea! I'll add it as therefore I've extracted repetitively used steps